cmd/juju/charmhub/infowriter.go

```diff
@@ -136,18 +139,19 @@ func (b bundleInfoWriter) Print() error {
 }
 
 type charmInfoOutput struct {
-	Name        string         `yaml:"name,omitempty"`
-	ID          string         `yaml:"charm-id,omitempty"`
-	Summary     string         `yaml:"summary,omitempty"`
-	Publisher   string         `yaml:"publisher,omitempty"`
-	Supports    string         `yaml:"supports,omitempty"`
-	Tags        string         `yaml:"tags,omitempty"`
-	Subordinate bool           `yaml:"subordinate"`
-	StoreURL    string         `yaml:"store-url,omitempty"`
-	Description string         `yaml:"description,omitempty"`
-	Relations   relationOutput `yaml:"relations,omitempty"`
-	Channels    string         `yaml:"channels,omitempty"`
-	Installed   string         `yaml:"installed,omitempty"`
+	Name        string                  `yaml:"name,omitempty"`
+	ID          string                  `yaml:"charm-id,omitempty"`
+	Summary     string                  `yaml:"summary,omitempty"`
+	Publisher   string                  `yaml:"publisher,omitempty"`
+	Supports    string                  `yaml:"supports,omitempty"`
+	Tags        string                  `yaml:"tags,omitempty"`
+	Subordinate bool                    `yaml:"subordinate"`
+	StoreURL    string                  `yaml:"store-url,omitempty"`
+	Description string                  `yaml:"description,omitempty"`
+	Relations   relationOutput          `yaml:"relations,omitempty"`
+	Channels    string                  `yaml:"channels,omitempty"`
+	Installed   string                  `yaml:"installed,omitempty"`
+	Config      map[string]charm.Option `yaml:"settings,omitempty"`
```

Comment:
Can we call this `config`..? The flag is called config, so you'd expect those to line up correctly. If we're expecting settings to be copied and pasted, can we do:

```yaml
config:
    settings:
        ...
```